AssetMappr/presentation/landingPage.py

```diff
@@ -11,19 +11,20 @@
 from dash import html
 import dash_bootstrap_components as dbc
 
+
 def makeLandingPage():
-    
-    return html.Div([
-            html.H1('Welcome to AssetMappr!', style={'color': '#2C3E50'}),
-            html.Br(),
-            html.H4('''We are a group of students that believe in the power of community participation. 
+
+    return html.Div(className='landing', children=[
```

Comment:
Maybe just add a quick comment here saying that className refers to the class defined in the .css file so it's clear